src/config.tmpl.h

```diff
@@ -0,0 +1,20 @@
+/* **************** OSW LABS ****************** */
+/**
+ * @license
+ * Copyright OSW Labs. All Rights Reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ */
+
+/* *************** AgroLab IoT **************** 
+ * Authors:  
+  - Sergio Morcuende (https://github.com/smorcuend)
+  - Rabindranath Andujar (https://github.com/ndujar)
+ * Source Code: https://github.com/OSWHackerspace/AgrolabIoT 
+ * Community: https://github.com/osweekends
+*/
+#define USERNAME ""
+#define DEVICE_ID ""
+#define DEVICE_CREDENTIAL ""
+
```

Comment:
We could add here indications for students as to what these variables are for.
/* ************************* AgroLab IoT **********************************
Modify these two variables to connect to your local wi-fi

SSID_1: Name of the network to connect to
SSID_PASSWORD_1: Password to access the network
*/